src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,11 +1,14 @@
-2018-04-18 23:35:28 UTC
+
+Type dsc to start/install the DS Consolidated Console
+
+2018-05-17 21:57:39 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: jugeorge
-Branch:      master
-Commit:      d08c7f54a125819caaa8c5f553206d1adbae60f9
+GitHub user: johnpaulkee
+Branch:      jobsSdk
```

Comment:
Code MUST be generated from a spec in the Azure master branch

Reply:
We did this in order to isolate this change from other APIs added to readme.md at the same time. 

It was just this change to readme: https://github.com/johnpaulkee/azure-rest-api-specs/commit/992e3dd8dfe722697d045458f827e9d554ef6ebb

Reply:
I'm not sure this is a good strategy. Either create and use a tag in the config file to scope the files you'd like to generate the code for or we could create a whitelisted branch in the repo to coordinate the work before merging with `psSdkJson6`
@shahabhijeet can you lend some thoughts too?